exchange/bidder.go

```diff
@@ -102,10 +102,22 @@ func adaptBidder(bidder adapters.Bidder, client *http.Client, cfg *config.Config
 		config: bidderAdapterConfig{
 			Debug:              cfg.Debug,
 			DisableConnMetrics: cfg.Metrics.Disabled.AdapterConnectionMetrics,
+			DebugInfo:          parseDebugInfo(debugInfo),
 		},
 	}
 }
 
+func parseDebugInfo(info *config.DebugInfo) config.DebugInfo {
```

Comment:
Minor point, but perhaps we just want to return the final boolean value here, rather than a full `config.DebugInfo`. Is there any benefit in keeping it packed inside of the the struct?

Reply:
Yes, fixed